src/ui/scss/components/image-gallery.scss

```diff
@@ -0,0 +1,13 @@
+.image-gallery {
+  h2
+  {
```

Comment:
Braces should be on the same line as the declaration with a single space before them. Preference admittedly varies here and some people like this style with braces on a new line, but the primary concern should always be consistency with the surrounding codebase.